deepinv/loss/augmentation.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+from typing import Union
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from deepinv.loss.loss import Loss
+from deepinv.loss.metric.metric import Metric
+from deepinv.physics.mri import MRI
+from deepinv.physics.forward import LinearPhysics
+from deepinv.physics.inpainting import Inpainting
+from deepinv.transform.base import Transform
+from deepinv.transform.rotate import Rotate
+from deepinv.transform.shift import Shift
+from deepinv.transform.augmentation import RandomPhaseError, RandomNoise
+
+
+class VORTEXLoss(Loss):
+    r"""Measurement data augmentation loss.
+
+    Performs data augmentation in measurement domain as proposed by
+    `VORTEX: Physics-Driven Data Augmentations Using Consistency Training for Robust Accelerated MRI Reconstruction <https://arxiv.org/abs/2111.02549>`_.
+
+    The loss is defined as follows:
+
+    :math:`\mathcal{L}(T_e\inverse{y,A},\inverse{T_i y,A T_e^{-1}})`
+
+    where :math:`T_1` is a random :class:`deepinv.transform.Transform` defined in k-space and
+    :math:`T_2` is a random :class:`deepinv.transform.Transform` defined in image space.
+    By default, for :math:`T_1` we add random noise :class:`deepinv.transform.RandomNoise` and random phase error :class:`deepinv.transform.RandomPhaseError`.
```

Comment:
```suggestion
    By default, for :math:`T_i` we add random noise :class:`deepinv.transform.RandomNoise`.
```

I would remove the default random phase error default since it only holds for MRI